src/orion/algo/mutate_functions.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+"""
+:mod:`orion.algo.mutate_functions` --
+Different mutate functions
+===========================================================================================
+
+.. module:: mutate_functions
+    :platform: Unix
+    :synopsis: Implement evolution to exploit configurations with fixed resource efficiently
+
+"""
+import logging
+
+import numpy as np
+
+logger = logging.getLogger(__name__)
+
+
+def default_mutate(self, winner_id, loser_id, multiply_factor, add_factor):
+    """Get a default mutate function"""
+    select_genes_key_list = np.random.choice(self.search_space_remove_fidelity,
+                                             self.eves.nums_mutate_gene,
+                                             replace=False)
+    self.copy_winner(winner_id, loser_id)
```

Comment:
I think we should extract the copying logic out of the mutate function. It is too specific to the EvolutionaryES algorithm.